composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "sabberworm/php-css-parser":"5.0.*"
+        "sabberworm/php-css-parser":"7.0.*"
```

Comment:
should be `^5.0|^6.0|^7.0`

Reply:
yes!, sorry ^7.0

Reply:
what about 6.0?

Reply:
6.0.x uses classes like Sabberworm\CSS\Value\String and it's not a valid php 7 name